src/sentry/api/endpoints/sentry_app_stats.py

```diff
@@ -22,9 +22,11 @@ def get(self, request, sentry_app):
 
         query_args = self._parse_args(request)
 
-        installations = SentryAppInstallation.with_deleted.filter(
-            sentry_app=sentry_app, date_added__range=(query_args["start"], query_args["end"])
-        ).values_list("date_added", "date_deleted", "organization_id")
+        installations = list(
```

Comment:
@nolachen I am a bit concerned about loading all the installations into memory instead of just the ones between `start` and `end`. We can get the total number of installs/uninstalls with a separate count query. Maybe we don't need to do anything now but maybe we could make a note of it.

Reply:
@scefali  Yeah I agree that this will potentially load a lot more into memory. But I don't know if it's better or worse to make 2 additional DB queries (1 for counting installs, 1 for counting uninstalls) as an alternative. Thoughts?

Reply:
@nolachen That's a good question. One advantage of the count query is that it doesn't have to retrieve the items but I can't say I know the performance of the DB well enough to say for sure which is worse. Maybe James would know.

Reply:
Could the date filter conditions be pushed into this query? Over time this query will incrementally get slower as more data accumulates. Two smaller bounded queries will do better than one unbounded query over time.

If you wanted to get fancy you could use `count(*), sum(if(date_deleted, 1, 0)` to get both install/uninstall counts in a single query however that will require grouping on either the date columns or organization_id.

Reply:
I updated the PR to have date filters in the main query that loads the objects into memory. This results in 2 extra count queries to count the total number of installs/uninstalls but ensures that only the objects that are needed are loaded into memory